src/container/Transfer/TransferPopup.tsx

```diff
@@ -58,10 +58,14 @@ const TransferPopup = ({
 
     const onSubmit = async (data: IForm) => {
         const { fee } = data
-        if (asset?.assetId === 'XCH') {
-            await sendXCHTx?.(address.address, amount, memo, fee)
-        } else {
-            await sendCATTx?.(address.address, asset, amount, memo, fee)
+        try {
+            if (asset?.assetId === 'XCH') {
```

Comment:
Dose XCH assetId is equal to  XCH in testnet?

Reply:
Yes, we set assetId manually, so it will always be `XCH`